src/config/yaml.ts

```diff
@@ -301,3 +301,14 @@ export const getRewardsConfig = () => {
 export const getAccountsConfig = (config = yamlConfig): AccountsConfig => ({
   initialStatus: config.accounts.initialStatus as AccountStatus,
 })
+
+export const getJwksArgs = (config = yamlConfig) => {
+  const urlJkws = config.oathkeeperConfig.urlJkws
+
+  return {
+    cache: true,
+    rateLimit: true,
+    jwksRequestsPerMinute: 5,
```

Comment:
should this be part of oathkeeperConfig ?

Reply:
do you mean exposing this to the yaml?

Reply:
yes